custom_components/frank_energie/coordinator.py

```diff
@@ -50,8 +59,13 @@ async def _async_update_data(self) -> dict:
         # Fetch data for today and tomorrow separately,
         # because the gas prices response only contains data for the first day of the query
         try:
-            prices_today = await self.api.prices(today, tomorrow)
-            prices_tomorrow = await self.api.prices(tomorrow, day_after_tomorrow)
+            if self.api.is_authenticated:
+                prices_today = await self.api.userPrices(today)
+                prices_tomorrow = await self.api.userPrices(tomorrow)
+            else:
+                prices_today = await self.api.prices(today, tomorrow)
+                prices_tomorrow = await self.api.prices(tomorrow, day_after_tomorrow)
```

Comment:
This would mean that pricedata behaves differently after updating. Would this be an issue?

It may be more accurate so it is only an improvement, but let's be clear in the release notes.

Reply:
Wat is het verschil tussen user prices en de normale market prices?

Reply:
> This would mean that pricedata behaves differently after updating. Would this be an issue?
> 
> It may be more accurate so it is only an improvement, but let's be clear in the release notes. What do you think, @bajansen ?

Wat is het verschil tussen user prices en de normale market prices?

Reply:
> > This would mean that pricedata behaves differently after updating. Would this be an issue?
> > It may be more accurate so it is only an improvement, but let's be clear in the release notes. What do you think, @bajansen ?
> 
> Wat is het verschil tussen user prices en de normale market prices?

User prices bevatten ook CO2 compensatie in het totale bedrag, dat geld niet voor de algemene prijzen.

Ik gok dat er ook een verschil zit tussen prijzen voor gebruikers met het oude / nieuwe modelcontract maar dat kan ik niet controleren. 

In mijn geval is het totaalbedrag uit deze call hetzelfde als het bedrag dat getoond word door de Frank App

Reply:
Verschil is voor mij ook enkel voor de gasprijs op dit moment, al zou dat in de toekomst natuurlijk kunnen veranderen.